ouroboros-network/test/Ouroboros/Network/BlockFetch/Examples.hs

```diff
@@ -128,7 +128,7 @@ blockFetchExample1 decisionTracer clientStateTracer clientMsgTracer
           decisionTracer clientStateTracer
           (sampleBlockFetchPolicy1 blockHeap currentChainHeaders candidateChainHeaders)
           registry
-          (BlockFetchConfiguration 2 1 10)
+          (BlockFetchConfiguration 2 1 10 0.01)
```

Comment:
I would use the named fields to set up the fields, e.g.
```
BlockFetchConfiguration {
    bfcMaxConcurrencyBulkSync = 2,
    bfcMaxConcurrencyDeadline = 1,
    bfcMaxRequestsInflight = 10,
    bfcDecisionLoopInterval = 0.01
  }
```
This way it's easier to figure out the meaning of each value.